salesagility/SuiteCRM
# Fix #9191 - Update antixss lib dependency

## Description




- Fix https://github.com/salesagility/SuiteCRM/issues/9191
- Current version of the lib was wrongly encoding special characters
- Update to antixss library fixes the problem

## Motivation and Context

- Fix https://github.com/salesagility/SuiteCRM/issues/9191

## How To Test This

- see: https://github.com/salesagility/SuiteCRM/issues/9191
- Need to set specific enconding
- Create records with special characters
- Check if the match character encoding done on previous versions 

## Types of changes


### Final checklist
